action.yml

```
@@ -35,12 +35,20 @@ inputs:
       Default is added.
     default: 'added'
     required: false
+  fail_level:
+    description: |
+      The level of failures that will cause the action to fail [any,info,warning,error]
+      Default is error.
+    default: ''
```

I think it makes sense to keep the default value the same as reviewdog. what do you think?
```suggestion
    description: |
      If set to `none`, always use exit code 0 for reviewdog. Otherwise, exit code 1 for reviewdog if it finds at least 1 issue with severity greater than or equal to the given level. Possible values: [none,any,info,warning,error] Default is `none`.
    default: 'none'
```

see also.
* https://github.com/reviewdog/reviewdog/pull/1854/files
* https://github.com/reviewdog/action-rubocop/pull/124

---

Thanks!